taskcluster/ci/config.yml

```diff
@@ -99,6 +99,12 @@ taskgraph:
             default-repository: https://github.com/mozilla-extensions/firefox-voice
             default-ref: prod
             type: git
+        dohresolverusagestudy:
+            name: "DoH Resolver Usage Study"
+            project-regex: doh-resolver-usage-study$
+            default-repository: git@github.com:mozilla-extensions/doh-resolver-usage-study
```

Comment:
This should be https://github.com/mozilla-extensions/...

With `git@`, we use ssh, and don't provide the key because `private-repo` isn't set. This is causing the clone to hang.